.github/workflows/snap.yaml

```diff
@@ -0,0 +1,32 @@
+name: Snap Testing
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+  # allow manual trigger
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Build and upload snap
+        id: build
+        # uses: canonical/edgex-snap-testing/build@v2
+        uses: canonical/edgex-snap-testing/build@main
+    outputs:
+      snap: ${{steps.build.outputs.snap}}
+
+  test:
+    needs: build
+    runs-on: ubuntu-latest
+    steps:
+      - name: Download and test snap
+        # uses: canonical/edgex-snap-testing/test@v2
+        uses: canonical/edgex-snap-testing/test@main
```

Comment:
It should be v2, as in the comment above.

Reply:
updated. the snap testing is still not triggered.

Reply:
This is because GA aren't active in this repo. They will be activated once this PR is merged.